app/api/models/recipes.js

```diff
@@ -0,0 +1,68 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db/db.js';
+import { User } from './users.js';
+
+export const Recipe = sequelize.define(
+    'recipes',
+    {
+        id: {
+            type: DataTypes.STRING(36),
+            primaryKey: true,
+            allowNull: false,
+        },
+        title: {
+            type: DataTypes.STRING(255),
+            allowNull: false,
+        },
+        category: {
+            type: DataTypes.STRING(255),
+            allowNull: false,
+        },
+        owner: {
+            type: DataTypes.STRING(36),
+            allowNull: false,
+            references: {
+                model: User,
+                key: 'id',
+            },
+            onDelete: 'CASCADE',
+        },
+        area: {
+            type: DataTypes.STRING(255),
+            allowNull: false,
+        },
+        instructions: {
+            type: DataTypes.TEXT,
+            allowNull: false,
+        },
+        description: {
+            type: DataTypes.TEXT,
+            allowNull: false,
+        },
+        thumb: {
+            type: DataTypes.STRING(255),
+            allowNull: false,
+        },
+        time: {
+            type: DataTypes.STRING(255),
+            allowNull: false,
+        },
+        ingredients: {
+            type: DataTypes.JSON,
+            allowNull: false,
+        },
+        created_at: {
+            type: DataTypes.BIGINT,
+            allowNull: false,
+        },
+        updated_at: {
+            type: DataTypes.BIGINT,
+            allowNull: false,
+        },
+    },
+    {
+        timestamps: false,
```

Comment:
`created_at` та `updated_at` треба видалити, нижче задати `timestamps: true`
```suggestion
        },
    },
    {
        timestamps: true,
```